stable/airflow/values.yaml

```diff
@@ -174,6 +174,14 @@ airflow:
   ## Run initdb when the scheduler starts.
   initdb: true
 
+  ## Additional scripts ran on the web container prior to startup
+  ## For example:
+  ## startupScript: |
+  #    echo 'adding connections'
+  #    && airflow connections -d --conn_id databricks_default
+  #    && airflow connections -a --conn_id databricks_default --conn_type databricks --conn_host company.cloud.databricks.com --conn_extra '{"host": "company.cloud.databricks.com", "token":"123"}' &&
```

Comment:
Not sure if it is a good recommendation to hardcode tokens here, secrets should be used instead

Reply:
I'd replace `"123"` in this example with `"$(cat "/some/secrets/mountpoint/token.secret")"` to show how a secret can be used. Be carefull with single and double quotes around the `$()` tough.